bedevere/bpo.py

```diff
@@ -37,6 +40,19 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
+            body = event.data["pull_request"]["body"]
+            if CLOSING_TAG not in body:
+                issue_number = issue_number_found.group("issue")
+                BPO_MSG = textwrap.dedent(f"""\
```

Comment:
Unfortunately, `textwrap.dedent()` doesn't work with f-strings (or there is some other reason I don't know) See https://github.com/python/bedevere/pull/27/files/dcc17d64f1c9a9d6b65703abbf697c132d76d785#r127503862 for the previous discussion.

You can see the example in my latest PR: https://github.com/python/cpython/pull/2807